src/main/java/org/openstreetmap/atlas/utilities/vectortiles/TippecanoeCommands.java

```diff
@@ -0,0 +1,146 @@
+package org.openstreetmap.atlas.utilities.vectortiles;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.artifact.versioning.DefaultArtifactVersion;
+import org.openstreetmap.atlas.exception.CoreException;
+import org.openstreetmap.atlas.utilities.runtime.RunScript;
+import org.openstreetmap.atlas.utilities.runtime.SingleLineMonitor;
+import org.openstreetmap.atlas.utilities.time.Time;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This utility class allows you to execute the tippecanoe command line tool ultimately via Java's
+ * runtime exec.
+ *
+ * @author hallahan
+ */
+public final class TippecanoeCommands
+{
+    private TippecanoeCommands()
+    {
+        // Utility class
+    }
+
+    private static final Logger logger = LoggerFactory.getLogger(TippecanoeCommands.class);
+    private static final SingleLineMonitor MONITOR = new SingleLineMonitor()
+    {
+        @Override
+        protected Optional<String> parseResult(final String line)
+        {
+            return Optional.of(line);
+        }
+    };
+
+    /**
+     * Allows you to check if you have installed tippecanoe on your system and if it satisfies the
+     * minimum version that we need.
+     *
+     * @return Whether you have a valid tippecanoe ready to execute in your environment.
+     */
+    public static boolean hasValidTippecanoe()
+    {
+        final String[] commandArray = new String[] { "tippecanoe", "--version" };
+        try
+        {
+            RunScript.run(commandArray, Collections.singletonList(MONITOR));
+        }
+        // When you look up the version, tippecanoe exits with 1, so getting here is normal.
```

Comment:
That is very strange behavior on `tippecanoe`'s part - so I appreciate this explanation. I initially passed over it and was confused as to why you were doing all this in the `catch` block.

It might be worth opening on issue for it on `tippecanoe`. Just a thought.